Source/JavaScriptCore/wasm/WasmIPIntGenerator.cpp

```diff
@@ -2650,8 +2650,8 @@ void IPIntGenerator::addCallCommonData(const FunctionSignature& signature, const
 
     Vector<uint8_t, 16> mINTBytecode;
     mINTBytecode.append(static_cast<uint8_t>(IPInt::CallArgumentBytecode::Call));
-    for (size_t i = 0; i < signature.argumentCount(); ++i) {
-        auto loc = callConvention.params[i].location;
+    for (auto argumentLocation : callConvention.params) {
```

Comment:
Should we take `argumentLocation` by reference? It's a 24 B struct.

Reply:
Sounds good, changed.